Cargo.toml

```diff
@@ -63,7 +63,7 @@ num-bigint-dig = "0.8"
 tracing = "0.1"
 rustls = { version = "0.21", features = ["dangerous_configuration"], optional = true }
 zeroize = { version = "1.6", features = ["zeroize_derive"] }
-tokio = { version = "1.32", features = ["time", "rt"], optional = true }
+tokio = { version = "1.32", features = ["time", "rt", "rt-multi-thread"], optional = true }
```

Comment:
Do we really need to enable the "rt-multi-thread" feature for `tokio`?

Reply:
`tokio::block_in_place` is guarded by check for `rt-multi-thread` feature. We can either enable it here or add another feature in `sspi` to and add it there (with `cfg` guard in code)

Reply:
Oh okay! In that case, I think it’s good as-is, let’s not add more complexity to this. Longer term, I also have a plan for removing the dependency to tokio completely and leave the user spawn a task in their runtime directly.